src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -163,6 +164,7 @@ public void run(ReaperApplicationConfiguration config, Environment environment)
 
     context.repairManager = RepairManager.create(
         context,
+        ClusterFacade.create(context),
```

Comment:
let's not inject a ClusterFacade into RepairManager (or RepairRunner or SegmentRunner or any other service class).

See comment in RepairRunner for how to.

Reply:
ok, done